main/politeness_logi.R

```diff
@@ -47,4 +46,4 @@ m_pol <- glm(label ~
 
 summary(m_pol)
 pR2(m_pol)
-write("Done. Results are stored in `politeness_logi.out`", stderr())
+write("Done. Results are stored in `bazel-bin/main/feed_data.runfiles/__main__/politeness_logi.out`", stderr())
```

Comment:
this sort of depends on where you run the script from. I don't even have this directory, because I'm building `bazel build //src:convo_politeness`. this file ends up in just bazel-bin for me.

Reply:
I've changed to use things like `getwd()` to print the absolute path of the file